invokeai/backend/install/invokeai_configure.py

```diff
@@ -654,10 +657,13 @@ def migrate_init_file(legacy_format: Path):
     old = legacy_parser.parse_args([f"@{str(legacy_format)}"])
     new = InvokeAIAppConfig.get_config()
 
-    fields = list(get_type_hints(InvokeAIAppConfig).keys())
+    fields = [x for x, y in InvokeAIAppConfig.__fields__.items() if y.field_info.extra.get("category") != "DEPRECATED"]
     for attr in fields:
         if hasattr(old, attr):
-            setattr(new, attr, getattr(old, attr))
+            try:
+                setattr(new, attr, getattr(old, attr))
+            except ValidationError as e:
+                print(f"* Ignoring incompatible value for field {attr}:\n  {str(e)}")
```

Comment:
Any reason we are manually setting attributes and not just instantiating the model with the fields it needs to have? eg `InvokeAIAppConfig(**args)`

Reply:
My rationale was to validate each field individually as it goes in and have invalid fields generate warnings rather than causing the whole initialization to fail.